src/main/java/com/amazonaws/services/dynamodbv2/AmazonDynamoDBLockClient.java

```diff
@@ -402,6 +402,14 @@ public static void createLockTableInDynamoDB(final CreateDynamoDBTableOptions cr
      */
     @SuppressWarnings("resource") // LockItem.close() does not need to be called until the lock is acquired, so we suppress the warning here.
     public LockItem acquireLock(final AcquireLockOptions options) throws LockNotGrantedException, InterruptedException {
+	return acquireLock(options, /*waitForLease=*/true);
+    }
+
+
+    /**
+     * @param waitForLease whether this method waits for at least the lease time before giving up
+     */
+    public LockItem acquireLock(final AcquireLockOptions options, final boolean waitForLease) throws LockNotGrantedException, InterruptedException {
```

Comment:
All arguments to acquireLock need to reside in AcquireLockOptions. Can you check to see if an option in that class might meet your need?